### PR TITLE
Variabilizes calico/node and calico-node

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Adds variables for calico/node container and running pod/service names to _config.yml
- Converts hard-coded instances of calico/node and calico-node to variables in master directory

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: master/getting-started/docker/installation/manual.md [hunk 1]
```diff
@@ -18,46 +18,47 @@ title: Installing Calico for Docker
    sudo chmod +x /usr/local/bin/calicoctl
    ```
 
-2. Configure access to your etcd cluster, [calicoctl - etcd datastore](/{{page.version}}/reference/calicoctl/setup/etcdv3).
-3. Launch `calico/node`:
+1. Configure access to your etcd cluster, [calicoctl - etcd datastore](/{{page.version}}/reference/calicoctl/setup/etcdv3).
```
Comment (doublek) [inline master/getting-started/docker/installation/manual.md]: Looks like `{{site.baseurl}}` is missing here?

File: master/getting-started/docker/installation/manual.md [hunk 2]
```diff
@@ -18,46 +18,47 @@ title: Installing Calico for Docker
...
+
+1. Launch `{{site.nodecontainer}}`:
 
    ```
    sudo calicoctl node run --node-image={{site.imageNames["node"]}}:{{site.data.versions[page.version].first.title}}
    ```
 
-Check that `calico/node` is now running:
+1. Check that `{{site.nodecontainer}}` is now running:
```
Comment (doublek) [inline master/getting-started/docker/installation/manual.md]: I think this should be `{{site.noderunning}}`.

File: master/getting-started/docker/installation/manual.md [hunk 3]
```diff
@@ -69,5 +70,5 @@ See [additional information on binding to an init system
 
 ## Next Steps
 
-With `calico/node` running, you are ready to start using {{site.prodname}} by following
+With `{{site.nodecontainer}}` running, you are ready to start using {{site.prodname}} by following
```
Comment (doublek) [inline master/getting-started/docker/installation/manual.md]: Should this be `{{site.noderunning}}`?

File: master/getting-started/docker/installation/vagrant-coreos/index.md [hunk 4]
```diff
@@ -64,24 +64,24 @@ And finally check that Docker is running on both hosts by running
...
-The Vagrant machines already have `calicoctl` installed. Use it to launch `calico/node`:
+The Vagrant machines already have `calicoctl` installed. Use it to launch `{{site.nodecontainer}}`:
 
     sudo ETCD_ENDPOINTS=http://172.17.8.101:2379 calicoctl node run --node-image={{site.imageNames["node"]}}:{{site.data.versions[page.version].first.title}}
 
 Append the `--use-docker-networking-container-labels` flag to the `calicoctl node run` command if you're combining
 [Docker Labels and {{site.prodname}} Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/security-using-docker-labels-and-calico-policy).
 
-Check that the `calico/node` container is running on this host:
+Check that the `{{site.nodecontainer}}` container is running on this host:
```
Comment (doublek) [inline master/getting-started/docker/installation/vagrant-coreos/index.md]: I think this should be `{{site.noderunning}}`, which is what we are checking for?

File: master/getting-started/docker/installation/manual.md (hunk 1 above)
Comment (emanic) [reply master/getting-started/docker/installation/manual.md]: This is purposeful, a relative link, which is more portable

File: master/getting-started/docker/installation/manual.md (hunk 2 above)
Comment (emanic) [reply master/getting-started/docker/installation/manual.md]: Yes, I think you are probably right.

File: master/getting-started/docker/installation/manual.md (hunk 3 above)
Comment (emanic) [reply master/getting-started/docker/installation/manual.md]: It does make more sense.

File: master/getting-started/kubernetes/installation/integration.md [hunk 5]
```diff
@@ -30,13 +30,13 @@ installation method.
 
 There are three components of a {{site.prodname}} / Kubernetes integration.
 
-- The {{site.prodname}} per-node docker container, [calico/node](https://quay.io/repository/calico/node?tab=tags)
+- The {{site.prodname}} per-node Docker container [`{{site.nodecontainer}}`
```
[the remark this answers is not in the document]
Comment (emanic) [reply master/getting-started/kubernetes/installation/integration.md]: Ooops!

**Comment by emanic** [comment]
Thanks @doublek !

**Pull request merged**